src/linter/ui5Types/fix/collections/jqueryFixes.ts

```diff
@@ -77,11 +77,20 @@ t.declareModule("jQuery", [
 				scope: FixScope.FirstChild,
 			})),
 		]),
+		t.namespace("resources", [
+			t.namespace("isBundle", callExpressionGeneratorFix({ // https://github.com/SAP/ui5-linter/issues/657
+				moduleName: "sap/base/i18n/ResourceBundle",
+				generator(_ctx, [moduleIdentifier], arg1) {
+					return `new ${moduleIdentifier}() instanceof ${arg1.trim()}`;
```

Comment:
I think this needs to be swapped (compare to the example in https://github.com/SAP/ui5-linter/issues/521#issuecomment-2875641949 - which is _somewhat_ correct but easy to misunderstand).

My understanding is that:

```js
const myBundle = ResourceBundle.create({url : "i18n/messagebundle.properties"});
if (jQuery.sap.resources.isBundle(myBundle)) {
    // ...
}
```

Should be migrated to:

```js
const myBundle = ResourceBundle.create({url : "i18n/messagebundle.properties"});
if (myBundle instanceof ResourceBundle) {
    // ...
}
```

In fact, `new ResourceBundle()` as currently implemented here throws an error `resource URL 'undefined' has unknown type`